frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -152,22 +154,53 @@ function PlotlyFigure({
         )
         spec.data = data
         spec.layout.selections = selections
-
-        const hasSelectedPoints: boolean = spec.data.some(
-          (trace: any) =>
-            "selectedpoints" in trace && trace.selectedpoints.length > 0
-        )
-        if (hasSelectedPoints) {
-          // make all other points opaque
-          spec.data.forEach((trace: any) => {
-            if (!trace.selectedpoints) {
-              trace.selectedpoints = []
-            }
-          })
+        if (spec.data) {
```

Comment:
just a small fix for when we try to access data but it doesn't exist for some reason... also a small fix for trying to access trace.selectedpoints when it doesn't exist.

Reply:
Are these fixes independent of the relayout aspects? If so, it might make sense to extract this part into a dedicated PR. Do you know why this might happen that `spec.data` is empty?

Reply:
They are. I can move this to a dedicated PR.